app/views/twoeye_content_review.scala.html

```diff
@@ -48,7 +48,9 @@ <h2 class="app-task-list__section">
         @for((page, index) <- review.pages.zipWithIndex) {
           <li class="app-task-list__item">
             <span class="app-task-list__task-name">
-              <a href="@{routes.TwoEyePageReviewController.onPageLoad(review.ocelotId, page.title.drop(1)).toString}" aria-describedby="page-@index">@page.title</a>
+              <a href="@{routes.TwoEyePageReviewController.onPageLoad(review.ocelotId, page.title.drop(1)).toString}" aria-describedby="page-@index">
+                  <span class="govuk-visually-hidden">@messages("2iReview.visuallyHiddenPage") @page.title</span>@page.title
+              </a>
```

Comment:
The additional text on the link causes the journey tests to fail, so you will have to chat to Roheel to workout whatto change